src/index.ts

```diff
@@ -18,4 +22,6 @@ export { default as FormGroup } from './FormGroup/index.js'
 export { default as Input } from './Input/index.js'
 export { default as Table } from './Table/index.js'
 export { default as Button } from './Button/index.js'
+export { default as Timeline } from './Timeline/index.js'
 export { default as UserIcon } from './UserIcon/index.js'
+export { CheckedIcon, InProgressIcon, PadlockIcon }
```

Comment:
Why not `export { default as CheckedIcon } from ...` and also, what's the reason I can't see an subsection called Icons here in the storybook ( or are those icons a special set of icons in which case they should be moved in the Timeline folder, maybe ? )

Reply:
this is fine